source/upgrade.txt

```diff
@@ -90,7 +90,7 @@ associated JIRA tickets for each release on `GitHub
 
 .. _php-v2-breaking-changes:
 
-Version 2.0 Breaking Changes
+Version 2.x Breaking Changes
```

Comment:
IMO, it makes sense to call these out as breaking changes in 2.0 - people will understand that these apply to future 2.x versions as well. This helps clarify that we don't make breaking changes in minor versions.

Reply:
Fixed! I also added a sentence in the "Breaking Changes" introduction noting that only major version releases have breaking changes